android/src/legacy/com/reactnativepagerview/PagerViewViewManager.kt

```diff
@@ -0,0 +1,173 @@
+                }
+            })
+
+            eventDispatcher.dispatchEvent(PageSelectedEvent(host.id, vp.currentItem))
+        }
+        host.addView(vp)
+        return host
+    }
+
+    override fun addView(host: NestedScrollableHost, child: View?, index: Int) {
+        PagerViewViewManagerImpl.addView(host, child, index)
+    }
+
+    override fun getChildCount(parent: NestedScrollableHost) = PagerViewViewManagerImpl.getChildCount(parent)
+
+    override fun getChildAt(parent: NestedScrollableHost, index: Int): View {
+        return PagerViewViewManagerImpl.getChildAt(parent, index)
+    }
+
+    override fun removeView(parent: NestedScrollableHost, view: View) {
+        PagerViewViewManagerImpl.removeView(parent, view)
+    }
+
+    override fun removeAllViews(parent: NestedScrollableHost) {
+        PagerViewViewManagerImpl.removeAllViews(parent)
+    }
+
+    override fun removeViewAt(parent: NestedScrollableHost, index: Int) {
+        PagerViewViewManagerImpl.removeViewAt(parent, index)
+    }
+
+    override fun needsCustomLayoutForChildren(): Boolean {
+        return PagerViewViewManagerImpl.needsCustomLayoutForChildren()
+    }
+
+    @ReactProp(name = "scrollEnabled", defaultBoolean = true)
+    fun setScrollEnabled(host: NestedScrollableHost, value: Boolean) {
+        PagerViewViewManagerImpl.setScrollEnabled(host, value)
+    }
+
+    @ReactProp(name = "initialPage", defaultInt = 0)
+    fun setInitialPage(host: NestedScrollableHost, value: Int) {
+        PagerViewViewManagerImpl.setInitialPage(host, value)
+    }
+
+    @ReactProp(name = "orientation")
+    fun setOrientation(host: NestedScrollableHost, value: String) {
+        PagerViewViewManagerImpl.setOrientation(host, value)
+    }
+
+    @ReactProp(name = "offscreenPageLimit", defaultInt = ViewPager2.OFFSCREEN_PAGE_LIMIT_DEFAULT)
+    operator fun set(host: NestedScrollableHost, value: Int) {
+        PagerViewViewManagerImpl.setOffscreenPageLimit(host, value)
+    }
+
+    @ReactProp(name = "overScrollMode")
+    fun setOverScrollMode(host: NestedScrollableHost, value: String) {
+        PagerViewViewManagerImpl.setOverScrollMode(host, value)
+    }
+
+    @ReactProp(name = "layoutDirection")
+    fun setLayoutDirection(host: NestedScrollableHost, value: String) {
+        PagerViewViewManagerImpl.setLayoutDirection(host, value)
+    }
+
+    override fun getExportedCustomDirectEventTypeConstants(): MutableMap<String, Map<String, String>> {
+        return MapBuilder.of(
+                PageScrollEvent.EVENT_NAME, MapBuilder.of("registrationName", "onPageScroll"),
+                PageScrollStateChangedEvent.EVENT_NAME, MapBuilder.of("registrationName", "onPageScrollStateChanged"),
+                PageSelectedEvent.EVENT_NAME, MapBuilder.of("registrationName", "onPageSelected"))
+    }
+
+    override fun receiveCommand(root: NestedScrollableHost, commandId: String?, args: ReadableArray?) {
```

Comment:
I think that ``commandId`` could be changed to ``commandName``. ``commandId`` suggests that function will expect some kind of ID, but name like ``setPage`` is being passed

Reply:
Since this is how codegen generates this function after internal discussion we decided to leave it as is.